public/manifest.json

```diff
@@ -26,5 +26,18 @@
       }
     }
   ],
-  "capture_links": "new_client"
+  "capture_links": "new_client",
+  "share_target": {
+    "action": "/",
```

Comment:
```suggestion
    "action": "/web-share-target",
```

We mint a fake URL here that only serves for this purpose.

Reply:
Got it!